agent/config/config.go

```diff
@@ -411,6 +411,7 @@ type CheckDefinition struct {
 	AliasService                   *string             `mapstructure:"alias_service"`
 	Timeout                        *string             `mapstructure:"timeout"`
 	TTL                            *string             `mapstructure:"ttl"`
+	H2PING                         *string             `json:"h2ping,omitempty" hcl:"h2ping" mapstructure:"h2ping"`
```

Comment:
```suggestion
	H2PING                         *string             `mapstructure:"h2ping"`
```

Minor: we removed the json and hcl struct tags here, since they haven't been used in some time.